Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.xaml

```diff
@@ -8,6 +8,7 @@
     </ResourceDictionary.MergedDictionaries>
```

Comment:
Just a heads up that @IbraheemOsama's Infinite Canvas uses the TextToolbar as well I believe, so not sure how these changes effect that (if at all).